src/heads-up/index.js

```diff
@@ -22,6 +22,8 @@ background.onMessage.addListener((message) => {
   }
 })
 
+store.commit('SET_THEME', { theme: getTheme() })
```

Comment:
Do we actually need the store for this? I know it was my suggestion :-/
Could also just import and calla `getTheme()` in `App.vue`. Maybe easier?

Reply:
Done and works